ligo/src/token/fa2.mligo

```diff
@@ -28,15 +28,15 @@ let check_sender (from_ , store : address * storage): address =
 let debit_from (amt, from_, token_id, store: nat * address * token_id * storage): storage =
   match Big_map.find_opt (from_, token_id) store.ledger with
     Some bal ->
-      let updated_store =
+      let store =
         match Michelson.is_nat (bal - amt) with
           Some new_bal ->
             { store with
               ledger = Big_map.update (from_, token_id) (Some new_bal) store.ledger
             }
         | None ->
             (failwith ("FA2_INSUFFICIENT_BALANCE") : storage)
-      in updated_store
+      in store
```

Comment:
Just wanted to mention that as @sras found out this can also be written like this:
```ocaml
    Some bal -> (
      match Michelson.is_nat (bal - amt) with
        Some new_bal ->
          { store with
            ledger = Big_map.update (from_, token_id) (Some new_bal) store.ledger
          }
      | None ->
          (failwith ("FA2_INSUFFICIENT_BALANCE") : storage)
    )
```
I don't think it's any better (or worst), I'm fine with it either way